src/pretalx/static/cfp/js/proposalTabTitles.js

```diff
@@ -0,0 +1,30 @@
+const titleParts = document.title.split("::");
+
+function updateTitle(newTitle) {
+    if (newTitle === "") {
+        document.title = titleParts.join(" :: ");
```

Comment:
You're splitting the title parts on `::`, but you're joining on ` :: ` – my naive reading would be that every title will contain double spaces once `updateTitle` is called. Probably not very visible, but with tab title space being at a premium … could you join on `::` here and remove the spaces next to `titleParts[1]` and `2` below?